src/components/facet-line/FacetLine.tsx

```diff
@@ -34,6 +34,11 @@ const FacetLine: React.FunctionComponent<FacetLineProps> = ({
 
   return (
     <>
+      {!data && (
+        <div className="ssc mt-48">
+          <div className="ssc-head-line" />
+        </div>
+      )}
```

Comment:
I know this is simple but to keep consistency we could consider placing this in a separate `FacetLineFiltersSkeleton` component.